python/ql/test/query-tests/Expressions/eq/NonPortableComparisonUsingIs.expected

```diff
@@ -1,2 +1 @@
 | expressions_test.py:51:4:51:11 | Compare | The result of this comparison with 'is' may differ between implementations of Python. |
-| expressions_test.py:56:4:56:16 | Compare | The result of this comparison with 'is' may differ between implementations of Python. |
```

Comment:
This looks to (now) be a false negative, but it doesn't look terribly serious. Is this right?

Reply:
That was arguably false positive and is now correct.
The assumption being that if an object is declared locally then an `is` comparison is OK as it is testing for identity. E.g.
```
SENTINEL = object()
...
if x is SENTINEL:
    ...
```